cli/command/stack/remove.go

```diff
@@ -63,7 +63,8 @@ func runRemove(dockerCli command.Cli, opts removeOptions) error {
 			return err
 		}
 		if versions.LessThan(version.APIVersion, "1.30") {
-			fmt.Fprintf(dockerCli.Err(), `WARNING: ignoring "configs" (requires API version 1.30, but the Docker daemon API version is %s)`, version.APIVersion)
+			fmt.Fprintf(dockerCli.Err(), `WARNING: ignoring "configs" (requires API version 1.30, but the Docker daemon API version is %s)
```

Comment:
Where's the `\n`... Oh!

Maybe ` + "\n"` or just just use `" .. \" ... \" " `?